panoptes_aggregation/panoptes/userify.py

```diff
@@ -0,0 +1,107 @@
+from collections import Iterable
+from json import dumps as jsonify
+from os import environ
+from panoptes_client import Panoptes, User
+import requests
+
+
+class ConfigurationError(Exception):
+    pass
+
+
+known_params = [
+    'destination'
+]
+
+connected = False
+
+users = {}
+
+destinations = {
+    'mast': 'https://mast-forward.zooniverse.org/',
+    'mockable': 'https://demo1580318.mockable.io/mast'
```

Comment:
No one can access this except for you, and it _technically_ has a 24 hour TTL, shouldn't be hardcoded here.

Reply:
Yeah it's... not ideal, but I'm not really sure how else to include a mockable endpoint we can use for testing without giving consumers to forward to arbitrary URLs, which I think would be a mistake. Any suggestions would be welcomed :)

Reply:
Would suggesting that it's internal with `_mockable` help any?

Reply:
I suggest removing it when this stops being a WIP. It's currently only useful in your personal dev env. If someone is developing locally then they can add this temporarily, but even then it doesn't seem like an actual request from you to Mockable needs to happen instead of looking at the logs and tests. 

I get that seeing it happen helps sanity check, this just shouldn't stay.

Reply:
I sort of wanted to keep it until we had the config stable on staging? But you're right that maybe removing it long-term is the best plan. If Zooniverse ever gets a paid plan it's easy enough to re-introduce

Reply:
The MAST URL is going to have to be updated at some point soon, so there's another PR in this feature's future anyway. For like $12/mo, that service is starting to seem like a no brainer, but even so its URL shouldn't be in a public repo.

I'd rather see settings like this somewhere outside of git. Maybe as a whitelist in a config file that matches against the `&destination=` param. Picking up changes would mean a redeploy rather than a PR.

Reply:
That's a great idea, especially since I already need to do that for the panoptes credentials! I'll start tinkering on that right away.

Reply:
so i have added dotenv support for fetching credentials and other secrets, and i have also added support for a yml file to specify endpoints